mobile/native/src/ln_dlc/node.rs

```diff
@@ -174,28 +173,32 @@ impl Node {
 
                     match reply_msg {
                         SubChannelMessage::Finalize(SubChannelFinalize { channel_id, .. }) => {
-                            let contract = self
+                            let contracts = self
                                 .inner
                                 .dlc_manager
                                 .get_store()
-                                .get_contract(&channel_id)
```

Comment:
too bad these were only type aliases, because such a API misuse seems to be quite common :(

Reply:
Definitely an insidious source of bugs. We could PRs on the relevant repos to introduce new types. I reckon `rust-dlc` would accept it.